client/src/components/layout/BuildForm.js

```diff
@@ -21,69 +21,68 @@ const BuildForm = (props) => {
 
     const [buildFormErrors, setBuildFormErrors] = useState({})
 
-    const [redirect, setRedirect] = useState(false)
-
-    const [newBuildID, setBuildID] = useState(0)
+    const [redirect, setRedirect] = useState({
+        redirectState: false,
+        newBuildID: 0
+    })
 
     const handleFormData = (event) => {
-        setNewBuild(
-            {
-                ...newBuild,
-                [event.currentTarget.name]: event.currentTarget.value
-            }
+        setNewBuild({
+            ...newBuild,
+            [event.currentTarget.name]: event.currentTarget.value
+        }
         )
     }
 
     const validateFormOnFrontEnd = () => {
```

Comment:
would be great to extract this out into its own file (possibly in the `services` folder) to reduce clutter in the component! (bonus, not required)

Reply:
Will circle back to this later in the day.